omod/pom.xml

```diff
@@ -0,0 +1,90 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+		 xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+		 xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+	<modelVersion>4.0.0</modelVersion>
+	<parent>
+		<artifactId>bahmni-events</artifactId>
+		<groupId>org.bahmni.module</groupId>
+		<version>1.0.0-SNAPSHOT</version>
+	</parent>
+
+	<artifactId>bahmni-events-omod</artifactId>
+	<packaging>jar</packaging>
+	<name>Bahmni Events omod</name>
+	<description>Bahmni module for events</description>
```

Comment:
Bahmni module for sending events on JMS

Reply:
Replaced "Bahmni module for sending events on JMS" in description